### DISCUSSION TITLE
_cfv enhancement - strikethrough

### DISCUSSION THREAD

**Discussion by Greenlight88** [cortexrd/Knack-Toolkit-Library#310] [Ideas]
Just a request for _cfv to support strikethrough - like a list or grid that includes, this has been dealt with stuff

**Comment by cortexrd** [comment]
Done!

Use the **"l"** option for **"Line-Through"**, which happens to be what CSS calls it and also a letter that was not already in use.

Its' been released a second ago in production: **v0.27.2**

Enjoy,
Norm

**Reply by Greenlight88** [reply]
whoah! that was fast - Thank you
